automation/render-local-env.sh

```diff
@@ -1,23 +1,84 @@
 #!/usr/bin/env bash
+set -e
+
+## Run from automation/
+## Clones the firecloud-automated-testing repo, pulls templatized configs, and renders them to src/test/resources
 
 # Defaults
 WORKING_DIR=$PWD
 VAULT_TOKEN=$(cat ~/.vault-token)
+FIRECLOUD_AUTOMATED_TESTING_BRANCH=master
 ENV=dev
 LOCAL_UI=false
+SERVICE=firecloud-ui
 
 # Parameters
 WORKING_DIR=${1:-$WORKING_DIR}
 VAULT_TOKEN=${2:-$VAULT_TOKEN}
+FIRECLOUD_AUTOMATED_TESTING_BRANCH=${3:-$FIRECLOUD_AUTOMATED_TESTING_BRANCH}
 ENV=${3:-$ENV}
 if [ "$4" = "local_ui" ]; then
   LOCAL_UI=true 
 fi
 
+confirm () {
+    # call with a prompt string or use a default
+    read -r -p "${1:-Are you sure?} [y/N] " response
+    case $response in
+        [yY])
+            shift
+            $@
+            ;;
+        *)
+            ;;
+    esac
+}
+
+# clone the firecloud-automated-testing repo
+clone_repo() {
```

Comment:
I wonder if this script should pull a fresh clone in a scratch space rather than side-effecting something outside of the clone of this repo. If we keep it like this, the README should describe this side effect so people are aware. For example, I'll likely have my own clone of firecloud-automated-testing that I might be working with and I wouldn't expect that to be changed by rendering test configs for UI.

Reply:
hm that's a good thought.  i could implement that pretty easily